client/driver/qemu.go

```diff
@@ -395,13 +462,13 @@ func (h *qemuHandle) Kill() error {
 	case <-h.doneCh:
 		return nil
 	case <-time.After(h.killTimeout):
+		h.logger.Printf("[DEBUG] driver.qemu - kill timeout exceeded")
```

Comment:
Can you include monitorPath and/or userPid in log messages so users have some way of connectin gthem to a particular VM?